Dockerfile

```diff
@@ -1,29 +1,17 @@
-# Build Geth in a stock Go builder container
-FROM golang:1.17-alpine as builder
+FROM golang:latest
```

Comment:
Shouldn't we bind to a specific Golang version.

Reply:
This is for development, I think that we can stick to latest so we can catch Go upgrade issues maybe from dev or CI